tests/datasets/test_transaction_processor.py

```diff
@@ -222,7 +222,9 @@ def build_result(self, meta: KafkaMessageMetadata) -> Mapping[str, Any]:
 
 class TestTransactionsProcessor(BaseTest):
     def __get_timestamps(slef) -> Tuple[float, float]:
-        timestamp = datetime.utcnow() - timedelta(seconds=5)
+        timestamp = datetime.utcnow().replace(tzinfo=timezone.utc) - timedelta(
```

Comment:
How it this different than

```suggestion
        timestamp = datetime.now(tz=timezone.utc) - timedelta(
```

Reply:
There isn't. I can change it.